Backend/Remora.Discord.API/API/Format/Markdown.cs

```diff
@@ -0,0 +1,119 @@
+//
+//  Markdown.cs
+//
+//  Author:
+//       Jarl Gullberg <jarl.gullberg@gmail.com>
+//
+//  Copyright (c) 2017 Jarl Gullberg
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+using JetBrains.Annotations;
+
+namespace Remora.Discord.API.Format
+{
+    /// <summary>
+    /// Provides helper methods to format strings into Discord Markdown.
+    /// </summary>
+    [PublicAPI]
+    public static partial class Markdown
```

Comment:
Are there any escape sequences we need to take into account for these?

Reply:
Is it the library's responsibility to take care of this? Personally, I don't think so. The API consumer should just provide the input and the library should add Discord's markdown around it.

Reply:
I'm kind of split on this as well. We can leave it for later :)